app/tests/tap-dance/3b-hold-int/native_posix.keymap

```diff
@@ -0,0 +1,12 @@
+#include <dt-bindings/zmk/keys.h>
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/kscan_mock.h>
+#include "../behavior_keymap.dtsi"
+
+&kscan {
+	events = <
+		ZMK_MOCK_PRESS(1,0,400)
+        ZMK_MOCK_PRESS(1,1,10)
+        ZMK_MOCK_RELEASE(1,1,10)
+	>;
```

Comment:
most tests include the release of all keys at the end - I'd add a `ZMK_MOCK_RELEASE(1,0,100)` at the end.